### ISSUE TITLE
Decrease default command/response stream length

### ISSUE THREAD

**Issue by dpipemazo** [elementary-robotics/atom#184]
I believe across C, C++ and python the default command/response stream length is 1024 entries which wastes a bunch of memory when we're passing around camera images. Shorten this to something reasonable like 100. This is tied to #183 since we're hitting our max mem limit until this is shortened. @plipay thoughts on appropriate length?

**Comment by Cody-G** [comment]
I see where to make these changes, just waiting on a suggestion for the new length.  I can do 128 since everybody loves powers of 2?

**Comment by Cody-G** [comment]
@dpipemazo in light of the new reference API do we want to reconsider this?

**Comment by dpipemazo** [comment]
@Cody-G agreed. With the new reference API we can close this.
